src/Microsoft.Health.Dicom.Core.UnitTests/Features/Store/StoreOrchestratorTests.cs

```diff
@@ -99,7 +99,7 @@ public async Task GivenFilesAreSuccessfullyStored_WhenStoringFile_ThenStatusShou
     public async Task GivenFailedToStoreFile_WhenStoringFile_ThenCleanupShouldBeAttempted()
     {
         _fileStore.StoreFileAsync(
-            Arg.Is<VersionedInstanceIdentifier>(identifier => DefaultVersionedInstanceIdentifier.Equals(identifier)),
+            Arg.Is<long>(identifier => DefaultVersionedInstanceIdentifier.Version.Equals(identifier)),
```

Comment:
I think you can just say:

```suggestion
            DefaultVersionedInstanceIdentifier.Version,
```

Reply:
Done